src/apps/frontend/components/typography/h2.tsx

```diff
@@ -1,7 +1,19 @@
-import React, { PropsWithChildren } from 'react';
+import React, { PropsWithChildren, CSSProperties } from 'react';
+import clsx from 'clsx'; 
 
-const H2: React.FC<PropsWithChildren> = ({ children }) => (
-  <h2 className="text-2xl font-bold text-black sm:text-title-xl2">
+interface H2Props extends PropsWithChildren {
+  style?: CSSProperties;
+  className?: string; 
+}
```

Comment:
can you explain the need of style and className to a specific typography component?